### PR TITLE
Proposal of PR template

### PR DESCRIPTION
**Pull request by carkod**
As discussed in a [previous catch-up #469](https://github.com/canonical-web-and-design/meeting-notes/issues/469) here I propose this PR template which is used across many of our websites. I created this PR based on our last discussion:

- Some repos, such as private repos, do not need an issue template, therefore we are not adding an issue template.
- However it seems all of our sites do have PRs, and sometimes we missed QA steps.
- This template includes some basic QA steps that are common across most projects.
- Other fields are optional, the user can still override and delete the placeholder content.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+## Done
+
+[List of work items including drive-bys]
+
+## QA
+
+- Check out this feature branch
+- Run the site using the command `./run serve`
+- View the site locally in your web browser at: http://0.0.0.0:8001/
```
Comment (anthonydillon) [inline .github/pull_request_template.md]: Maybe we should change the port to be clear it should be replaced

**Review by anthonydillon — APPROVED** [review]
LGTM :+1:

**Comment by anthonydillon** [comment]
Once addressed LGTM :+1:

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+## Done
+
+[List of work items including drive-bys]
+
+## QA
+
+- Check out this feature branch
+- Run the site using the command `./run serve`
+- View the site locally in your web browser at: http://0.0.0.0:port/ (replace port with the correct port in your [project](https://canonical-web-and-design.github.io/practices/project-structure/ports.html))
```
Comment (anthonydillon) [inline .github/pull_request_template.md]: I was thinking something similar to http://0.0..0.0:{port}/ which should be similar to the port in the (`./env` file)[https://github.com/canonical-web-and-design/website-boilerplate/blob/master/.env].

**Review by anthonydillon — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md (hunk 2 above)
Comment (carkod) [reply .github/pull_request_template.md]: Makes sense. Updated

**Pull request merged**